eng/native/configurecompiler.cmake

```diff
@@ -480,6 +479,12 @@ if (CLR_CMAKE_HOST_UNIX)
 
   # We mark the function which needs exporting with DLLEXPORT
   add_compile_options(-fvisibility=hidden)
+  
+  # Separate functions so linker can remove them. But not on tvOS because
+  # -ffunction-sections is not supported with -fembed-bitcode.
+  if (NOT CLR_CMAKE_HOST_TVOS)
+    add_compile_options(-ffunction-sections)
+  endif()
```

Comment:
Is tvOS still compiled with bitcode? I thought Apple basically deprecated that across the board.

https://developer.apple.com/documentation/Xcode-Release-Notes/xcode-14-release-notes

Reply:
CI is using v13:

> Running 'CC="/Applications/Xcode_13.2.1.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang"..

Reply:
I don't think it necessarily matters what CI uses. The change is that App Store doesn't require it anymore for new submissions.

Reply:
https://github.com/dotnet/runtime/pull/79349